app/helpers/admin/analytics_helper.rb

```diff
@@ -8,5 +8,24 @@ def track_analytics_data(category, action, label)
         "track-label" => ActionController::Base.helpers.strip_tags(label),
       }
     end
+
+    def track_analytics_data_on_load(title)
+      {
+        event_name: "page_view",
+        page_view: {
+          government_department_name: "Government Digital Service",
+          access_level: access_based_roles(current_user&.permissions),
+          publishing_app: "Whitehall",
+          user_created_at: current_user&.created_at&.to_date,
+          document_type: title,
+        },
+      }.to_json
+    end
+
+    def access_based_roles(permissions)
```

Comment:
@minhngocd I am actually wondering if we should just send "Admin"/"GDS Admin"/"Editor"?